pkg/client/uuid.go

```diff
@@ -0,0 +1,276 @@
+package client
+
+import (
+	"bytes"
+	"database/sql/driver"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"reflect"
+	"time"
+
+	"github.com/jackc/pgtype"
+)
+
+// Code came from: https://github.com/jackc/pgtype/blob/75446032b914bb0be5e07da29c976034c0a666cf/uuid.go#L3-L230
+
+type UUID struct {
```

Comment:
Would embedding the original/calling the original code

Reply:
Great suggestion! Fixed!